{{cookiecutter.package_name}}/.github/CODEOWNERS

```diff
@@ -1,2 +1,2 @@
 # default owners
-* @albrja @beatrixkh @hussain-jafari @mattkappel @ramittal @rmudambi @stevebachmeier
+* @albrja @hussain-jafari @mattkappel @patricktnast @rmudambi @stevebachmeier
```

Comment:
Maybe we ought to add a new line here since this will be used in all future research model repos.

Reply:
Oops, I see you merged, nvm!